prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md

```diff
@@ -0,0 +1,7 @@
+Interesting thing to note: instead of writing coral reefs as arrecifes de coral, this paper wrote arrecifes coralinos.
```

Comment:
Does this have a slightly different connotation to you?

Reply:
No, I just wanted to note the differences we might see when referring to coral reefs in texts.